requirements-minimal.txt

```diff
@@ -3,7 +3,7 @@ jsonschema>=3.2.0
 PyYAML>=5.4
 scipy>=1.4.1
 h5py>=3.9.0
-hdmf>=3.11.0
+hdmf @ git+https://github.com/hdmf-dev/hdmf.git@dev
```

Comment:
Aren't we waiting for this to be merged?

Reply:
I assume you mean 'released' - I will poke the HDMF team to get one out

It's for https://github.com/hdmf-dev/hdmf/pull/1013, the essential fix to even make it possible to set the data IO for table columns

Reply:
Yes, correct, I meant released. All right.